tests/cloudpickle_test.py

```diff
@@ -2511,6 +2511,10 @@ def inner_function():
         sys.version_info < (3, 9),
         reason="Can cause CPython 3.8 to segfault",
     )
+    @pytest.mark.skipif(
+        sys.version_info > (3, 14),
+        reason="Can cause CPython 3.14 to segfault",
+    )
```

Comment:
Could you please add a link to the upstream issue to make it easier to assess when this skip marker can safely be removed?